.azure-pipelines/ci-build.yml

```diff
@@ -17,288 +17,300 @@ pool:
   vmImage: windows-latest
 
 variables:
-  buildPlatform: 'Any CPU'
-  buildConfiguration: 'Release'
+  buildPlatform: "Any CPU"
```

Comment:
Any chance the change in indentation across the file is intended? Are you able to confirm this does not break the pipeline?